### DISCUSSION TITLE
Why does AUTO_BACKUP_PAUSE_WITH_NO_PLAYERS require PUBLIC=1 ?

### DISCUSSION THREAD

**Discussion by Ricket** [mbround18/valheim-docker#761] [Q&A]
Title. I was using `lloesche/valheim-server-docker` docker image until now, and had a non-public server and it was not making backups while no players are online. So it seems like it's possible.

**Comment by mbround18** [comment]
Your sever status is dependent on whether or not we can poll it via the steamcmd. The valheim server instance does not provide server commands to get the current players. If they do, we can alter this behavior to work with public 0

**Reply by Ricket** [reply]
https://github.com/lloesche/valheim-server-docker/blob/d79d54ec33a15daf56b170b0d02600d87deaefa9/common#L160

looks like the other docker image uses `nstat` to get the UDP network activity and if it's greater than a threshold, then a player is online... clever solution, maybe you can copy it?

I'm trying to find a better valheim solution because I am having issues with that one where the server will stop accepting connections / be in a stuck dead state after an update. but, it's a small server with just some friends, so I don't want my server public, and I definitely don't want backups while nobody's online - which is most of the time.

**Reply by mbround18** [reply]
I can look into this method
